src/main.c

```diff
@@ -2408,6 +2412,23 @@ void handle_mouse_input() {
     }
 }
 
+
+float setJump(int megaJump, float dy)
```

Comment:
It would be cleaner if the modified code for "mega jumping" was just in the corresponding pull request & not mixed up in this pull request which is just as its title says "Adding palm trees to world".